helm/orca/config/alerts-mapping.yaml

```diff
@@ -185,6 +185,416 @@ prometheus:
         kind: node
         properties:
           name: instance
+    - name: KubeCronJobRunning
+      source_mapping:
+        origin: kubernetes
+        kind: cronjob
+        properties:
+          name: cronjob
+          namespace: namespace
+    - name: KubeJobCompletion
+      source_mapping:
+        origin: kubernetes
+        kind: job
+        properties:
+          name: job_name
+          namespace: namespace
+    - name: KubeJobFailed
+      source_mapping:
+        origin: kubernetes
+        kind: job
+        properties:
+          name: job_name
+          namespace: namespace
+    - name: KubeAPIDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeSchedulerDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeControllerManagerDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeCPUOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeMemOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeMemoryOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeCPUQuotaOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
+        properties:
+          name: namespace
+    - name: KubeMemoryQuotaOvercommit
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
+        properties:
+          name: namespace
+    - name: KubeQuotaExceeded
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
+        properties:
+          name: namespace
+    - name: KubeVersionMismatch
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: KubeStateMetricsDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: AlertmanagerDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: NodeExporterDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: PrometheusDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: PrometheusOperatorDown
+      source_mapping:
+        origin: kubernetes
+        kind: cluster
+        properties: {}
+    - name: TargetDown
+      source_mapping:
+        origin: kubernetes
+        kind: namespace
```

Comment:
Is it possible to map this alert to service kind instead of namespace kind based on `service` + `namespace` [labels](https://github.com/coreos/kube-prometheus/blob/a8b4985de4dc8582dcaf8fd43d6f88a0c08c463d/manifests/prometheus-rules.yaml#L1728)?